terraform/instances.tf

```diff
@@ -102,13 +102,14 @@ resource "null_resource" "api_provision" {
   provisioner "remote-exec" {
     inline = [
       "chmod +x /tmp/install_base_packages.sh",
+      "sudo /tmp/install_base_packages.sh",
       "sudo mv /home/ubuntu/hab-sup.service /etc/systemd/system/hab-sup.service",
       "sudo systemctl daemon-reload",
       "sudo systemctl start hab-sup",
       "sudo systemctl enable hab-sup",
-      "sudo hab svc load habitat/builder-api --group ${var.env} --bind router:builder-router.${var.env} --strategy at-once --url ${var.bldr_url} --channel ${var.release_channel}",
-      "sudo hab svc load habitat/builder-api-proxy --group ${var.env} --bind http:builder-api.${var.env} --strategy at-once --url ${var.bldr_url} --channel ${var.release_channel}",
-      "sudo hab svc load core/sumologic --group ${var.env} --strategy at-once --url ${var.bldr_url} --channel ${var.release_channel}",
+      "sudo hab svc load habitat/builder-api --force --group ${var.env} --bind router:builder-router.${var.env} --strategy at-once --url ${var.bldr_url} --channel ${var.release_channel}",
+      "sudo hab svc load habitat/builder-api-proxy --force --group ${var.env} --bind http:builder-api.${var.env} --strategy at-once --url ${var.bldr_url} --channel ${var.release_channel}",
+      "sudo hab svc load core/sumologic --force --group ${var.env} --strategy at-once --url ${var.bldr_url} --channel ${var.release_channel}",
```

Comment:
Why `--force`? Presumably these commands are being run on fresh instances, right?

Reply:
unless one fails, then you end up in fail hell

Reply:
Wouldn't you taint those resources and start fresh? `--force` would only come into play when there's already a spec file for that package present.

Reply:
Yeah @christophermaier has a good point.. if we're in a bad state on initial provision, we'd want to re-create that instance until we could deploy clean.

Reply:
iirc Terraform should be idempotent and convergent so I think these changes make sense.

Reply:
Best to follow the "Nuke it from orbit... it's the only way to be sure" school of infrastructure management here.

Reply:
Yeah that's interesting since we're using a null resource for provisioning. The null resource is tainted but it never taints the VM. I'll double back and see if we can change the resource type to.make this safer